src/main/apollon-editor.ts

```diff
@@ -90,24 +91,26 @@ export class ApollonEditor {
    * @param options options to change the export behavior (add margin, exclude element ...)
    * @param theme the theme which should be applied on the svg
    */
-  static exportModelAsSvg(
+  static async exportModelAsSvg(
     model: Apollon.UMLModel,
     options?: Apollon.ExportOptions,
     theme?: DeepPartial<Styles>,
-  ): Apollon.SVG {
-    const div = document.createElement('div');
+  ): Promise<Apollon.SVG> {
+    const container = document.createElement('div');
+    const root = createRoot(container);
     const element = createElement(Svg, { model, options, styles: theme });
-    const svg = render(element, div);
-    const { innerHTML } = div;
-    unmountComponentAtNode(div);
+    const svg = new Svg({ model, options, styles: theme });
+    root.render(element);
+    await delay(500);
```

Comment:
I think this should be some kind of waiting function for rendering, a hardcoded delay is a bit unclean.

Reply:
I suggest replacing this with `await delay(0)` and adding a commend on why you need to wait for one render cycle.